src/ai/configuration.hpp

```diff
@@ -99,8 +99,8 @@ class configuration {
 
 
 	/**
-	 * @param[in] cfg the config to be read
-	 * @param[out] parsed_cfg parsed config
+	 * @param[in] original_cfg the config to be read [out] parsed config
```

Comment:
That bit on the end (`[out] parsed config`) doesn't look like it belongs now? Particularly since this is a const reference.